src/components/ActivityCard/ActivityCard.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { useState } from "react";
+import { Pressable, StyleSheet, Text, View } from "react-native";
+import { Badge, Icon } from "react-native-elements";
+import useCurrentAuthUser from "../../hooks/useCurrentAuthUser";
+import { refetchGetTripQuery } from "../../screens/Itinerary/types/getTripQuery";
+import * as Types from "../../types.d";
+import { useCreateActivityReactionMutation } from "./types/CreateActivityReactionMutation";
+import { useRemoveActivityReactionMutation } from "./types/RemoveActivityReactionMutation";
+
+export interface ActivityCardProps {
+    id: string;
+    name: string;
+    date: Types.Maybe<string> | undefined;
+    activityReactions: Array<{
+        id: string;
+        activityReactionType: Types.ActivityReactionType;
+        addedByUser: { id: string };
+    }>;
+    tripId: string;
+}
+
+export default function ActivityCard(props: ActivityCardProps) {
```

Comment:
Missing return type

Reply:
@DominicLuidold we could make that explicit in the eslint config? (mark it as an error instead of warning)

Reply:
Yeah, I'll have a look at it once this PR is merged.